### ISSUE TITLE
Draft In-Client Specs

### ISSUE THREAD

**Issue by ZeldaZach** [Cockatrice/Cockatrice#3012]
So my thought on how to draft in Cockatrice.

1) On the new game room, an option will be for "draft". If this option is selected, instead of loading a normal game, it will open a special tab.

2) This room will `iframe` in a dr4ft.com draft with the appropriate configurations, set by the room host. 

3) Players who join this room will have the same URL auto-open in tab, and they will draft as normal using the program.

4) Once the draft completes, dr4ft will send the deck to the user automatically via API call. The deck will be saved to the Downloads folder and then the Deck Editor will open up the deck. 

Curious what people think about this plan. This would be an all local to the client change as dr4ft works well without having a server watching over it and they have their own anti-cheat measures in place.

**Comment by ancestral** [comment]
Will the game send the user name over to the website so players know who they are drafting with? Also, if the app crashes the draft is done, will the user still get their deck?

**Comment by ZeldaZach** [comment]
Yeah, I'll work on an API with dr4fters to accept and send commands with Cockatrice to help make games better. If the app crashes, not much I can do really... Maybe have a backup plan to do something?

**Comment by tooomm** [comment]
linking https://github.com/Cockatrice/Cockatrice/issues/1855

**Comment by tooomm** [comment]
[dr4ft](https://github.com/dr4fters/dr4ft) added an API in https://github.com/dr4fters/dr4ft/pull/96 some time ago. Documentation can be found [here](https://github.com/dr4fters/dr4ft/blob/master/doc/api.md).

**Comment by tooomm** [comment]
As a first step, does it makes sense to simply `iframe` to the dr4ft.info page in a new tab and the user goes from there without leaving the app?
`Menu --> Open dr4ft page` or simply "dr4ft". Like there is an option to open the deck editor in a new tab.

The more advanced integration can come later.
